doc.go

```diff
@@ -0,0 +1,83 @@
+// Generated by running
+//  go generate github.com/gonum/matrix
+// DO NOT EDIT.
+//
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// Package matrix provides common error handling mechanisms for matrix operations
+// in mat64 and cmat128.
+//
+// Overview
+//
+// This section provides a quick overview of the matrix package. The following
+// sections provide more in depth commentary.
+//
+// matrix provides:
+//  - Error type definitions
+//  - Error recovery mechanisms
+//  - Common constants used by mat64 and cmat128
+//
+// BLAS and LAPACK
```

Comment:
Okay. 

The problem I want solved is that the current size of the documentation makes it seem like the package does much more than it does. This might be the first thing users see, so it does need to direct them to the right place, but otherwise should just say what it does.